tests/api/search/entities.test.js

```diff
@@ -84,18 +84,20 @@ describe('search:entities', () => {
         _.map(results, 'uri').should.containEql(work.uri)
       })
 
-      // FIX: The reason this test fails seems to be that we use
-      // the autocomplete analyzer with a max_gram lower those terms, as you can
-      // see by searching 'Metaphysis Anfangsgrü der Naturwisse' instead.
-      // The test also passes when search_analyzer is let unspecified at indexation
-      // letting the search query use the autocomplete tokenizer
-      xit('should find a label containing long terms', async () => {
+      it('should ignore the case', async () => {
+        const label = "L'EAU DOUCE EN PÉRIL"
+        const work = await createWork({ labels: { fr: label.toLowerCase() } })
+        await waitForIndexation('entities', work._id)
+        const results = await search({ types: 'works', search: label, lang: 'fr', exact: true })
+        _.map(results, 'uri').should.containEql(work.uri)
+      })
+
+      it('should find a label containing terms longer than the autocomplete max ngram', async () => {
+        // Current max_ngram=10
         const label = 'Metaphysische Anfangsgründe der Naturwissenschaft'
```

Comment:
since `max_ngram` is now a constant in `server/db/elasticsearch/settings/settings.js`, what about requiring it inside this test and generate a label with words longer than this limit instead.
```suggestion
        const long_word = () => randomWords(10).split(' ').join('').slice(0, max_ngram + 1)
        const label = `${long_word()} ${long_word()} ${long_word()}` 
```

Reply:
done in 869d0fe5